src/main/scala/org/broadinstitute/clio/webservice/oidcHeaders.scala

```diff
@@ -0,0 +1,77 @@
+package org.broadinstitute.clio.webservice
+
+import akka.http.scaladsl.model.headers.{ModeledCustomHeader, ModeledCustomHeaderCompanion}
+
+import scala.util.Try
+
+
+/**
+  * An OIDC access token with value {@param token}.
```

Comment:
This scaladoc syntax isn't valid. Check the output of `sbt doc` in `target/scala-2.12/api/org/broadinstitute/clio/webservice/OidcAccessToken.html`. Perhaps just use:

> An OIDC access token with value \`token\`.

Same goes for other similar usages below.